qpmodel/DataType.cs

```diff
@@ -390,7 +399,6 @@ public override List<Expr> AllColumnsRefs(bool refresh = false)
                 // make a coopy of selection list and replace their tabref as this
                 var r = new List<Expr>();
 
-                Debug.Assert(query_.bounded_);
```

Comment:
before Bind(), AllColumnRefs() does not return right result?

Reply:
unexpect change reverted.